grails-app/controllers/com/mini/asaas/payer/PayerController.groovy

```diff
@@ -76,7 +76,7 @@ class PayerController {
 
     def update() {
         try {
-            Long customerId = 1
+            Long customerId = getCurrentUser().customer.id
```

Comment:
outro ponto, se vc for usar somente em um local esse customerId, não cria variável... Se for usar em mais de um lugar dai convém, mas se não, já chama direto onde vc vai uasar